### ISSUE TITLE
Pull Request:  (sindresorhus/awesome) or (qinwf/awesome-R)

### ISSUE THREAD

**Issue by grabear** [grabear/awesome-rshiny#9]
Where should we put the awesome-rshiny list? @sdhutchins @daattali @dpastoor @ramnathv @ucg8j 

- [x] Under [Front-End Development](https://github.com/sindresorhus/awesome#front-end-development)?
- [x] As a sub bullet under the R programming language?
- [ ] Merge with [awesome-R](https://github.com/qinwf/awesome-R#awesome-)
- [x] Or add a link to [awesome-R](https://github.com/qinwf/awesome-R#awesome-)
- [ ] Or should we do something completely different?

**Comment by ucg8j** [comment]
My vote would be 

- [x]  Under Front-End Development?
This may help increase awareness outside of the R community.
- [ ]  As a sub bullet under the R programming language?
Not sure what you are referring to, sounds good! Could you please provide a link to clarify?
- [ ]  Merge with awesome-R
No, I think shiny is big enough to warrant it's own awesomeness. I had a quick look at [awesome-python](https://github.com/vinta/awesome-python)... and the [awesome-flask](https://github.com/humiaozuzu/awesome-flask) and [awesome-django](https://github.com/rosarior/awesome-django). They each have their own.
- [x]  Or add a link to awesome-R
That would definitely be helpful. Happy to help out with PRs if the group decides this way. It's also what [awesome-python](https://github.com/vinta/awesome-python#web-frameworks) does.
- [ ]  Or should we do something completely different?
I'll let the more imaginative come up with other suggestions!

**Comment by grabear** [comment]
Ok good deal!  @ucg8j This might make it clearer.

- [Awesome Lists](https://github.com/sindresorhus/awesome)      
    - [ ] Under [Front-End Development](https://github.com/sindresorhus/awesome#front-end-development)?
**OR**
    - [ ] As a sub bullet under the R programming language?
- Awesome-R
    - [ ] Merge with [awesome-R](https://github.com/qinwf/awesome-R#awesome-)
**OR**
    - [x] Or add a link to [awesome-R](https://github.com/qinwf/awesome-R#awesome-)
- Or should we do something completely different?
    - [X] @sdhutchins already made [this](https://github.com/jnv/lists/pull/102/files) Pull Request

So we could do one of the first options and one of the second options.

**Comment by ucg8j** [comment]
Cool! I've just submitted a [pull request on awesome-R](https://github.com/qinwf/awesome-R/pull/134). There's quite a lot of open PRs so that might take a while. It looks like @sdhutchins PR has been accepted.

Ah, I think it just clicked as to what you meant. Add on this awesome list, a link to the awesome-R page?

**Comment by grabear** [comment]
Not exactly.  Sorry I can see how that was confusing.  What I meant by 'Merge with Awesome-R' was to basically add an r-Shiny section to their list.  So we copy and paste all of our README.md into their README.md.  But I don't think that will be useful after all. @ucg8j

**Comment by grabear** [comment]
Got another [PR](https://github.com/sindresorhus/awesome/pull/1069) going.  @ucg8j @sdhutchins @daattali @dpastoor @jcheng5

**Comment by sdhutchins** [comment]
Everything looks _awesome_ and uh _err_ _shiny_ @ucg8j @grabear

**Comment by grabear** [comment]
So @ucg8j's PR went through on [Awesome-R](https://github.com/qinwf/awesome-R/pull/134)!
Cheers! @sdhutchins @dpastoor

**Comment by sdhutchins** [comment]
Can we go ahead and close this out? @grabear

**Comment by grabear** [comment]
Not yet.  The other PR is still open.  I'll look at the guidelines for the main awesome list to make sure we comply.

**Comment by sdhutchins** [comment]
Didn't even realize it was still open. Hmmm.
